src/utils/env/client.ts

```diff
@@ -45,3 +45,59 @@ export function getGaId() {
 export function getSquidUrl() {
   return checkEnv(process.env.NEXT_PUBLIC_SQUID_URL, 'NEXT_PUBLIC_SQUID_URL')
 }
+
+export function getFirebaseApiKey() {
```

Comment:
for this, because all of them seems related to each other, I think we can refactor this to just a single function, like
getFirebaseConfig() that returns all of the config

the example is like getRedisConfig in src/utils/env/server